integration/service/oasis/stationfinder/tnsearchfinder/low_energy_location_station_finder_test.go

```diff
@@ -0,0 +1,68 @@
+package tnsearchfinder
+
+import (
+	"reflect"
+	"sync"
+	"testing"
+
+	"github.com/Telenav/osrm-backend/integration/service/oasis/stationfinder/stationfindertype"
+)
+
+func TestLowEnergyLocationStationFinderIterator1(t *testing.T) {
+	sf := createMockLowEnergyLocationStationFinder1()
+
+	var wg sync.WaitGroup
+	wg.Add(1)
+	go func(wg *sync.WaitGroup) {
```

Comment:
Nit: It's a simple sync funciton, the `go func() ...` and `waitgroup` seems redundant.

Reply:
do you mean using a `channel` to wait for return value instead?

Reply:
No. The test function only includes a `go func()`. I mean, is it really need the async?

Reply:
Copy that, test function could be run in different go-routines, if just use one `go func()` then might be no need addition `no`